plugins/snapshots/erofs/erofs.go

```diff
@@ -108,6 +130,16 @@ func NewSnapshotter(root string, opts ...Opt) (snapshots.Snapshotter, error) {
 		return nil, err
 	}
 
+	if config.enableDmverity {
```

Comment:
Just my own opinion here,
I guess dm-verity enabling can be on the per snapshot/layer basis?
no need to check dmverity here unless the dmverity is mandatory.

or the boolean should be `dmverity_mode=[auto/off/on]` instead `enable_dmverity` as the differ, because only `force_on` should fail if some layer doesn't enable dmverity.